### ISSUE TITLE
files: show terminal status: stopped/running, process title

### ISSUE THREAD

**Issue by alon** [sagemathinc/cocalc#3907]
I find myself in the following situation:
1. I created 5 different terminals.
2. One of them has a running vim session.
3. somehow, the tabs have been closed (maybe I did and forgot)
4. I want to get back to the vim session without checking all 5 terminals.

I can see the vim via "ps | grep vim". Optimally, I would have some notice about the title of the window in the files tab. At minimum it will tell me which terminal is "live".

Sorry if this is a duplicate - I went through "terminal" title bugs (3 pages!) and didn't see from the title anything looking similar.

**Comment by haraldschilly** [comment]
I don't think this is a duplicate. The closest equivalent I can think of is a way to see all running jupyter notebook sessions, and then be able to open up those which are currently running (and not go through all files). My first thought is to have a panel "Currently running ..." in the project settings (say), and it gives you a link to click on to open the corresponding file.

The files tab could have a column about the status of a session associated with a file. The problem here is, this could be spread across many directories and you might have a hard time finding the file. Still, it would be neat to see there as well which files are "active".
